tests/antithesis/docker-compose.yml

```diff
@@ -46,10 +73,29 @@ services:
       ETCD_INITIAL_CLUSTER_TOKEN: "etcd-cluster-1"
       ETCD_INITIAL_CLUSTER: "etcd0=http://etcd0:2380,etcd1=http://etcd1:2380,etcd2=http://etcd2:2380"
       ETCD_INITIAL_CLUSTER_STATE: "new"
+      ETCD_DATA_DIR: "/var/etcd/data"
+    user: "${USER_ID:-root}:${GROUP_ID:-root}"
+    depends_on:
+      init:
+        condition: service_completed_successfully
     ports:
       - 32379:2379
+    volumes:
+      - ${ETCD_ROBUSTNESS_DATA_PATH:-/tmp/etcddata}2:/var/etcd/data
 
   client:
     image: 'etcd-client:latest'
     container_name: client
     entrypoint: ["/opt/antithesis/entrypoint/entrypoint"]
+    user: "${USER_ID:-root}:${GROUP_ID:-root}"
```

Comment:
This causes `Failed to save visualization: failed to visualize, err: open history.html: permission denied`